extension-compose/src/main/java/com/mapbox/maps/extension/compose/MapboxMap.kt

```diff
@@ -23,18 +45,27 @@ public fun MapboxMap(
   modifier: Modifier = Modifier,
   /**
    * Defines the initialisation configurations for a [MapboxMap].
+   *
    * It can only be set once and not mutable after the initialisation. Mutating the [MapInitOptions]
    * during recomposition will result in a [IllegalStateException].
    */
   mapInitOptions: MapInitOptions = MapInitOptions(LocalContext.current.applicationContext),
+  /**
+   * Callback to be invoked when the user clicks on the map view.
+   */
+  onMapClickListener: OnMapClickListener = OnMapClickListener { false },
```

Comment:
Is it clear for the developers that to ignore clicks they need to set a listener that returns `false`?

Would it be easier to understand if we allow nullable listeners?

Reply:
We would also save some microseconds from `GesturesPluginImpl.handleClickEvent` since the listener would be empty 😝

Reply:
yeah, makes sense, we can make it default to null.